app/assets/javascripts/map.js

```diff
@@ -193,7 +193,6 @@
 
           if (App.Map.validCoordinates(coordinates)) {
             marker = createMarker(coordinates.lat, coordinates.long, coordinates.title);
-            marker.options.id = coordinates.investment_id;
```

Comment:
In the commit "Update old usages of investments JSON data", you reference commit 3fa3c90db. Since that commit, the instance method json_data in app/models/map_location.rb appears to be unused.

Should we remove it in this commit, or leave it for later cleanup?